src/schema/search/index.ts

```diff
@@ -171,17 +181,20 @@ export const Search: GraphQLFieldConfig<void, ResolverContext> = {
           args,
           {
             arrayLength: totalCount,
-            sliceStart: pageOptions.offset,
+            sliceStart: offset,
           }
         )
 
+        const pageInfo = connection.pageInfo
+        pageInfo.endCursor = pageToCursor(page + 1, size)
```

Comment:
Is this what you mean by needing to calculate this manually? Will this manual process need to be propagated to other fields as well? Would it be possible to hide this behind some kind of factory function?

Reply:
Yea for sure. But, this is just our initial use case of it. I think it's premature to start abstracting it away when we don't have it yet working in this more explicit way in production.

All in good time @damassi ! I like to let things settle and think about things, have to use it in a different context, before I can settle on a proper abstraction.